### DISCUSSION TITLE
Can't render & in README. WHY??????

### DISCUSSION THREAD

**Discussion by CompetitiveLin** [community/community#41337] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

`$$dp[i] = i $$` works:

$$dp[i] = i $$
Not for `$$dp[i] = i \&\& dp[i+1] = i+ 1$$`:
$$dp[i] = i \&\& dp[i+1] = i+ 1$$
WHY????

**Comment by jfullstackdev** [comment, marked as answer]
```
$$dp[i] = i \\&\\& dp[i+1] = i+ 1$$
```

this looks like this, 

$$dp[i] = i \\&\\& dp[i+1] = i+ 1$$

is that the correct one ? I just added another slash for each escape

**Reply by CompetitiveLin** [reply]
seems right, but `\\&` can't be rendered as $\\&$ in vscode and [here](https://saxarona.github.io/mathjax-viewer/). I don't know why..
![image](https://user-images.githubusercontent.com/55691019/206891778-a7d6b9e0-9509-4661-9b20-ab175040a90a.png)

**Reply by jfullstackdev** [reply]
yup, that's very common thing in Markdown, there is no standard so every app will have a different render
